shapeshift/web
# feat: create multi-hop swapper slice

## Description

Progression of multi-hop trade capability. This PR includes the creation of multi-hop swappers slice and RTK query api, along with a couple of react hooks for the multi-hop trade UI.

Includes some minor refactoring of the lifi swapper to enable calling from RTK query.

## Pull Request Type


## Issue (if applicable)

NA

## Risk

Moderate risk of broken lifi trades and quotes.

## Testing

- Check lifi quotes are working as expected
- Check lifi trades are working as expected

### Engineering

See above

### Operations

The only user-facing changes are related to lifi.

## Screenshots (if applicable)
